app/assets/stylesheets/components/_filter-section.scss

```diff
@@ -0,0 +1,56 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-c-filter-section {
+  border-bottom: 1px solid $govuk-border-colour;
+  margin: govuk-spacing(4) 0;
+
+  &:last-child {
+    border-bottom: 0;
+    margin-bottom: 0; 
+  }
+
+  &:first-child {
+    margin-top: 0; 
+  }
+
+  &[open] .app-c-filter-section__summary::before {
+    transform: translateY(70%) rotate(-45deg);
```

Comment:
Might be worth using [`prefixed-transform` mixin](https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/mixins/_prefixed-transform.scss#L1) from Publishing Components